### DISCUSSION TITLE
Is backporting to 1.18.2 Fabric planned? Hoping for bugfixes

### DISCUSSION THREAD

**Discussion by kbsmilk** [Ladysnake/Requiem#613] [General]
Mostly interested in this one fix, actually: "Fix underwater mobs going zoom with AdditionalEntityAttributes"

https://github.com/Ladysnake/Requiem/commit/156b13bbd9c4d97805c4d3b09df9780a0a1a29b6

**Comment by Redfan2** [comment]
Hi, is this still relevant to you, given it has been some time since you asked it.
